multitenancy/tenant-manager-api/src/main/java/io/apicurio/multitenant/api/TenantsResourceImpl.java

```diff
@@ -70,7 +70,7 @@ public Response createTenant(NewRegistryTenantRequest tenantRequest) {
         tenant.setName(tenantRequest.getName());
         tenant.setDescription(tenantRequest.getDescription());
         tenant.setCreatedOn(new Date());
-        tenant.setCreatedBy(null); //TODO extract user from auth details
+        tenant.setCreatedBy(tenantRequest.getCreatedBy());
```

Comment:
so fleet-manager will be responsible for passing the user who created the tenant?

Reply:
Yes, since we're integrating them using service accounts, the principal here will be the name of the service account, not the name of the user who requested the tenant. We need to pass that info from the fleet manager.